web/themes/custom/pineapple/sass/base/forms/_admin-feedback-form.scss

```diff
@@ -0,0 +1,13 @@
+.block-admin-feedback {
+  #feedback-message {
+    margin-top: $unit *3;
+
+    .admin-feedback-form {
+      padding: 0;
+    }
+  }
+
+  h2 {
+    margin-top: $unit *3;
```

Comment:
Remove this please it's not doing anything.

Reply:
It is adding top spacing on the thank you message.

Reply:
The other margin top in this file is adding that top spacing.

Reply:
fairs.